.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2.1
 upload-coverage: &upload-coverage
   run:
     name: Upload test coverage to Codecov
-    command: bash <(curl -s https://codecov.io/bash)
+    command: bash <(curl --retry 10 --retry-delay 5 --retry-max-time 60 -s https://codecov.io/bash)
```

Comment:
Why the max time? If it's already specified 10 times with 5 seconds in between. I guess if it is because of possible timeouts, it should be longer

Reply:
Because you can have it fail because of timeout. And I do not want to wait longer than 60 seconds ... I expect it to succeed fast or just fail. Waiting for more time is not really a good idea. Fast feedback is better